www/model/errors.ts

```diff
@@ -86,6 +86,7 @@ export class ServerUnreachable extends RegularNativeError {}
 export class IllegalServerConfiguration extends RegularNativeError {}
 // TODO: Seems like a candidate for RedFlagNativeError; only used by Android?
```

Comment:
It is used by both Android and Apple to signal that the plugin failed to establish the VPN tunnel. We should make it a RedFlagNativeError.

Reply:
Done.